### PR TITLE
manifest: fix whitespace issues

### PR DESCRIPTION
**Pull request by gauravsitlani**
**Description of your changes:**

**Which issue is resolved by this Pull Request:**
Resolves #

**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/latest/Contributing/development-flow/#commit-structure)).
- [ ] **Skip Tests for Docs**: If this is only a documentation change, add the label `skip-ci` on the PR.
- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/latest/Contributing/development-flow/#submitting-a-pull-request)
- [ ] [Pending release notes](https://github.com/rook/rook/blob/master/PendingReleaseNotes.md) updated with breaking and/or notable changes for the next minor release.
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.

### CODE REVIEW

**Comment by galexrt** [comment]
Removing skip-ci label as the manifests are partially used for tests and anyways it shouldn't hurt.

File: deploy/examples/cluster.yaml [hunk 1]
```diff
@@ -154,75 +154,75 @@ spec:
...
+  #    mon:
+  # Monitor deployments may contain an anti-affinity rule for avoiding monitor
+  # collocation on the same node. This is a required rule when host network is used
+  # or when AllowMultiplePerNode is false. Otherwise this anti-affinity rule is a
+  # preferred rule with weight: 50.
+  #    osd:
+  #    prepareosd:
+  #    mgr:
+  #    cleanup:
+    annotations:
```
Comment (galexrt) [inline deploy/examples/cluster.yaml:183]:
```suggestion
  annotations:
```

**Review by galexrt — CHANGES_REQUESTED** [review]
Let's clean up the spacing in the re-indented lines as well so they can be easily uncommented and used.

File: deploy/examples/cluster.yaml (hunk 1 above)
Comment (gauravsitlani) [reply deploy/examples/cluster.yaml]: updated.

**Review by galexrt — APPROVED** [review]

**Comment by galexrt** [comment]
One thing that I saw looking further up in the `cluster.yaml` file is that some commented-out fields are done like this `#fieldName:` while others are `# fieldName:`.

I would "vote" for `#fieldName:` as enabling a section is done by removing one `#` character.

@travisn @BlaineEXE what are your thoughts?

**Review by travisn — APPROVED** [review]
I'd vote to keep it as in this PR. Only the # should need to be removed on each line, and it seems more readable to keep the # characters aligned instead of immediately next to the text at different indentation on each line.

**Pull request merged**